python-avd/pyavd/_eos_designs/structured_config/underlay/prefix_lists.py

```diff
@@ -118,7 +113,11 @@ def ipv6_prefix_lists(self: AvdStructuredConfigUnderlayProtocol) -> None:
         sequence_numbers = EosCliConfigGen.Ipv6PrefixListsItem.SequenceNumbers()
         for index, network in enumerate(collapse_addresses(get_ipv6_networks_from_pool(self.shared_utils.loopback_ipv6_pool)), start=1):
             sequence_numbers.append_new(sequence=index * 10, action=f"permit {network} eq {self.inputs.fabric_ip_addressing.loopback.ipv6_prefix_length}")
-        if self.shared_utils.overlay_vtep and self.shared_utils.underlay_ipv6_numbered:
+        if (
+            self.shared_utils.overlay_vtep
+            and self.shared_utils.underlay_ipv6_numbered
+            and not (self.shared_utils.node_config.vtep_loopback and self.shared_utils.node_config.vtep_loopback == "Loopback0")
```

Comment:
I would have expected this?
```suggestion
            and self.shared_utils.vtep_loopback.lower() != "loopback0"
```

Reply:
Fixed.